.golangci.yml

```
@@ -70,10 +70,12 @@ linters-settings:
 # Disable goconst in test files, often we have duplicated strings across tests, but don't make sense as constants.
 issues:
   exclude-rules:
-    - path: (_test\.go|utilities_testing\.go)
+    # cover utilities_testing.go and _test.go files
+    - path: (_test.*go)
```

I think the `.go` suffix still needs escaping with `\.go`

```suggestion
    - path: (_test.*\.go)
```

---

These suggestions/fixes have been lost in the force push